docs/user_guide/05_model_architecture.md

```diff
@@ -60,23 +60,25 @@ class Model():
 ## Match torchvision formats
 
 From this definition we can see three format requirements. The model must be able to take in a batch of images in the order [channels, height, width]. The current model weights are trained on 3 band images, but you can update the check_model function if you have other image dimensions.
-The second requirement is that the model ouputs a dictionary with keys ["boxes","labels","scores"], the boxes are formatted following torchvision object detection format. From the [docs](https://pytorch.org/vision/main/models/generated/torchvision.models.detection.retinanet_resnet50_fpn.html#torchvision.models.detection.retinanet_resnet50_fpn)
+The second requirement is that the model outputs a dictionary with keys ["boxes","labels","scores"], the boxes are formatted following torchvision object detection format. From the [docs](https://pytorch.org/vision/main/models/generated/torchvision.models.detection.retinanet_resnet50_fpn.html#torchvision.models.detection.retinanet_resnet50_fpn)
 
-```{note}
+.. note::
```

Comment:
Looks like MyST callouts use `:::{note}` and `:::` https://mystmd.org/guide/admonitions (but I could be missing something since we just made this change)